doc/topics/tutorials/writing_tests.rst

```diff
@@ -452,6 +452,61 @@ can be used
                )
 
 
+Test Groups
+===========
+Salt has four groups
+* fast - Tests that are ~10s or faster. Fast tests make up ~75% of tests and can run in 10 to 20 minutes.
+* slow - Tests that are ~10s or slower.
+* core - Tests of any speed that test the root parts of salt.
+* flaky-jail - Test that need to be temporarily skipped.
+
+
+Pytest Decorators
+* @pytest.mark.slow_test
+* @pytest.mark.core_test
+* @pytest.mark.flaky_jail
+
+.. code-block:: python
+    @pytest.mark.core_test
+    def test_ping(self):
+        """
+        test.ping
+        """
+        self.assertTrue(self.run_function("test.ping"))
+
+You can also mark all the tests in file.
+
+.. code-block:: python
+    pytestmark = [pytest.mark.core_test]
```

Comment:
```suggestion
    pytestmark = [pytest.mark.core_test,]
```
And let black docs formar it.

Reply:
Question: will the black docs formatting work for the version that appears here on GitHub, rendered in GitHub flavored rST?

Reply:
I have a strong dislike for that `,`

Reply:
> I have a strong dislike for that `,`

Yes, I noticed.

Vertical code is easier to read.

Reply:
Never mind! I just remembered this one is the one in the core Salt docs. Not the contributing guide. Disregard my question.

Reply:
> Question: will the black docs formatting work for the version that appears here on GitHub, rendered in GitHub flavored rST?

Black will format the code blocks like it does for salt code, doesn't do any rendering.

Reply:
> I have a strong dislike for that `,`

Once black formats it, it'll look nicer.